openfe/protocols/openmm_rfe/_rfe_utils/relative.py

```diff
@@ -229,6 +232,164 @@ def __init__(self,
         self._omm_hybrid_topology = self._create_hybrid_topology()
         logger.info("Hybrid system created")
 
+    @staticmethod
+    def _verify_cmap_compatibility(
+            cmap_old: openmm.CMAPTorsionForce | None,
```

Comment:
Could not account for the `None`? Also, indentation is odd here, did ruff formatting pass?

Reply:
Alternatively, just don't check for None in the method that calls this method.